src/app/reservation/create-reservation/floor-and-wing-step/floor-and-wing-step.component.html

```diff
@@ -23,6 +23,8 @@ <h4 class="text-2xl font-extrabold">Vleugel:</h4>
     </mat-form-field>
     <div class="w-full flex justify-center items-center space-x-4">
       <button class="border-2 border-primary-dark-purple rounded-2xl w-fit h-fit px-3 py-0.5 text-2xl text-primary-dark-purple" matStepperPrevious>Vorige</button>
-      <button class="bg-primary-dark-purple rounded-2xl w-fit h-fit px-3 py-1 text-2xl text-white" matStepperNext>Volgende</button>
+      <button class="bg-primary-dark-purple rounded-2xl w-fit h-fit px-3 py-1 text-2xl text-white"
+              (click)="checkValidReservationType()"
```

Comment:
kun je de validatie niet beter via de form doen. Je kunt namelijk met forms kijken of ze valid zijn of niet met form.valid en daarmee bijvoorbeeld een popup of andere melding weergeven